src/app/public/modules/link-list/types/link.ts

```diff
@@ -0,0 +1,12 @@
+import { NavigationExtras } from '@angular/router';
+
+export type SkyLink = {
```

Comment:
We will need JSDocs here in order to fully populate the docs.

Example: https://github.com/blackbaud/skyux-lookup/blob/master/src/app/public/modules/country-field/types/country.ts